analyzer/src/main/kotlin/curation/ClearlyDefinedPackageCurationProvider.kt

```diff
@@ -196,15 +196,9 @@ private fun SourceLocation?.toArtifactOrVcs(): Any? =
             }
 
             else -> {
-                val url = sourceLocation.url ?: run {
-                    when (sourceLocation.provider) {
-                        // TODO: Implement provider-specific mapping of coordinates to URLs.
-                        else -> ""
-                    }
-                }
-
+                // TODO: Implement provider-specific mapping of coordinates to URLs.
```

Comment:
I believe the `TODO` should go above line 201 now, which will also make it look nicer in the following change.